### PR TITLE
Print debug message for http cookiefile used during git clone

### PR DESCRIPTION
**Pull request by chaodaiG**
We use http cookiefile for authenticating during cloning with http protocol, the cookiefile is rotated by either https://github.com/kubernetes/test-infra/blob/master/prow/cmd/grandmatriarch or https://github.com/istio/test-infra/tree/master/authentikos. When the cloning failed for any reason, half of the time the suspect tured to be staled cookiefile being used, but there is no immediate easy way for us to figure this out. Adding this debug line would help greatly in this scenario as grandmatriarch already prints md5sum.

### CODE REVIEW

**Review by alvaroaleman — APPROVED** [review]

File: prow/clonerefs/run.go
Comment (fejta) [inline prow/clonerefs/run.go:92]: What is the path that we cannot read?

File: prow/clonerefs/run.go
Comment (fejta) [inline prow/clonerefs/run.go:94]: Info() seems more appropriate here.

**Review by fejta — COMMENTED** [review]
/hold

File: prow/clonerefs/run.go
Comment (chaodaiG) [reply prow/clonerefs/run.go:92]: line 89 includes the path in logger

File: prow/clonerefs/run.go
Comment (chaodaiG) [reply prow/clonerefs/run.go:94]: Already merged. I can create another PR updating this if still desired

Comment (fejta) [reply prow/clonerefs/run.go:94]: No need,LGTM!

**Pull request merged**
